doc/api/fs.md

```diff
@@ -4719,6 +4573,92 @@ The following constants are meant for use with the [`fs.Stats`][] object's
   </tr>
 </table>
 
+## File System Flags
+
+The following flags are available wherever the `flag` option takes a
+string:
+
+* `'a'` - Open file for appending.
```

Comment:
It seems in all the list items all the lines should be aligned with the first line minus list bullet and space, like this:
```md
* `'a'` - Open file for appending.
  The file is created if it does not exist.
```
This may change the wrapping within 80 character limit.

Reply:
This should be taken care of as well :)